core/plannernotes.c

```diff
@@ -135,6 +179,26 @@ void add_plan_to_notes(struct diveplan *diveplan, struct dive *dive, bool show_d
 				"<th style='padding-left: 10px; float: left;'>%s</th></tr></thead><tbody style='float: left;'>",
 				translate("gettextFromC", "gas"));
 	}
+	for (icyl=0; icyl<MAX_CYLINDERS; icyl++)	// If dive plan has an OC cylinder with helium, then initialise ICD table:
+		if ((dive->cylinder[icyl].cylinder_use == OC_GAS) && (get_he(&dive->cylinder[icyl].gasmix) > 0)) {	
+			istrimix = true;
+			icdlen = 0;
+			icdlen += snprintf(icdbuffer + icdlen, sz_icdbuf - icdlen,
+					"<table><tbody style='float: left;'><tr><td colspan=5>%s:</td></tr>",
```

Comment:
single tabs instead of double tabs should be used here for the second line of `snprintf(...)`.
same for the rest of the calls bellow.

Reply:
done